docs/pub_sub_with_mqtt.md

```diff
@@ -0,0 +1,65 @@
+## Publish message using Mosquitto broker to another device
+
+To be able to publish a message to another device subscribed to a particular topic;
+ 
+ 1. Install the [mosquitto broker](https://www.vultr.com/docs/install-mosquitto-mqtt-broker-on-ubuntu-20-04-server/) in your machine, to be able to use the commands available 
+
+ 2. If publishing to another device, ensure the `TwinklyMaha` application is started on that device. This will be the device subscribed to the topic.
+
+ Run:
+ ```shell
+ $ cd TwinklyMaha
+
+ $ iex -S mix
+
+ ```
+
+ Before starting the application ensure you have the following exported in your system 
+
+ ```
+  export CLIENT_ID=:sfractal2020
+  export MQTT_HOST="test.mosquitto.org"
+  export MQTT_PORT=1883
```

Comment:
Aren't we using application.get_env? And these variables are in the `host.exs` file

Reply:
We only applied that to BlinkyMaha

Reply:
okay makes sense